src/main/java/org/apache/commons/lang3/math/Fraction.java

```diff
@@ -31,7 +31,8 @@
  * based and thus suffers from various overflow issues. For a BigInteger based
  * equivalent, please see the Commons Math BigFraction class. </p>
  *
- * @deprecated for fraction implementation from commons-numbers
+ * @deprecated for <a href="https://commons.apache.org/proper/commons-math/javadocs/api-3.6.1/index.html">Fraction</a>
+ * from commons-numbers
```

Comment:
Still not quite right: You talk about Commons Numbers but link to Commons Math as a whole. You want the URL to the Commons Numbers Fraction package or class.

Reply:
Aw, sorry, resolved that moment already